flake8_quotes.py

```diff
@@ -1,8 +1,11 @@
+import os
 import tokenize
 
 import pep8
 
-__version__ = '0.2.2'
+
+with open(os.path.join(os.path.dirname(__file__), 'VERSION')) as handle:
```

Comment:
Doing this overrides the default of `'rt'`, which may be preferable in this case:

> Python distinguishes between files opened in binary and text modes, even when
> the underlying operating system doesn't. Files opened in binary mode
> (appending 'b' to the mode argument) return contents as bytes objects without
> any decoding. In text mode (the default, or when 't' is appended to the mode
> argument), the contents of the file are returned as strings, the bytes having
> been first decoded using a platform-dependent encoding or using the specified
> encoding if given.

Do you still want to explicitly pass `'r'` here? We're working with a text file, and it's probably better to have the interpreter explicitly perform decoding for us, instead of implicitly trusting Python to do the right then when asked to `strip()` some bytes.